api/versionedsignedproposal.go

```diff
@@ -14,7 +14,8 @@
 package api
 
 import (
-	apiv1deneb "github.com/attestantio/go-eth2-client/api/v1/deneb"
+	"errors"
+	"github.com/attestantio/go-eth2-client/api/v1/deneb"
```

Comment:
Please can we keep this import as `apiv1deneb`?  This helps to differentiate spec structs from those in the API.